modules/mini-games/Race.lua

```diff
@@ -73,11 +73,10 @@ Global.register({
     laps = tbl.laps
     gate_boxes = tbl.gate_boxes
     start_players = tbl.start_players
-    
 end)
 
 local function reset_table(table)
-    for i, value in pairs(table) do
+    for i, _ in pairs(table) do
         table[i] = nil
     end
```

Comment:
This can be `for i in pairs` but really it should be `for k in pairs` or `for key in pairs` since this might not be an index because you are using pairs. General rule: i or index is used with ipairs, k or key is used with pairs.

Reply:
but then you can argue it shood be i cause i might not be a key but an index (ill change it though)